### PR TITLE
Use batch instead of multiple requests in cloud storage backend

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: cliquet/storage/cloud_storage.py [hunk 1]
```diff
@@ -183,29 +183,51 @@ def get_all(self, resource, user_id, filters=None, sorting=None,
...
-                                        headers=self._build_headers(resource))
-                resp.raise_for_status()
-                for record in resp.json()['items']:
+                params_ = list(params)
+                params_ += [("%s%s" % (FILTERS[op], k), v)
+                              for k, v, op in filters]
+                querystring = '&'.join(['%s=%s' % (p, v) for p, v in params_])
+                batch_payload['requests'].append({
+                    'path': url + '?%s' % querystring,
+                })
```
Comment (ametaireau) [inline cliquet/storage/cloud_storage.py]: Neat! I'm actually thinking we could add a context manager here and when we exit() from it we send the batch request.

**Comment by leplatrem** [comment]
In order to fix the tests on Travis, we should make sure the underlying Kinto runs the cliquet version of the current branch.
This is not the case, it uses the last pypi version :(

**Comment by leplatrem** [comment]
@Natim @ametaireau r?

**Comment by Natim** [comment]
> In order to fix the tests on Travis - I think we should use only one venv for everything and finish the install by ``pip install -e .`` so that we will use the same version of cliquet for everything (kinto and tests).

**Pull request merged**